config/routes.rb

```diff
@@ -76,6 +76,8 @@
       concerns :exportable
     end
 
+    get 'articles/:id/ris' => 'articles#show', as: :articles_ris, constraints: ->(req) { req.format = :ris }
```

Comment:
Had to do this with a crazy constraint instead of `{defaults: { format : ...` so that the right mimetype gets set and picked up